### PR TITLE
done some alert dialogs, postDriver, etc

### PR DESCRIPTION
**Pull request by Shikhar-Pandey**

### CODE REVIEW

File: lib/functions/getDataFromApi.dart [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import 'package:http/http.dart' as http;
+import 'package:liveasy/models/truckModel.dart';
+import 'dart:convert';
+
+class GetDataFromApi {
```
Comment (liveasy97) [inline lib/functions/getDataFromApi.dart:5]: change name of class and file to more descriptive

File: lib/functions/getDriverDetailsFromDriverApi.dart [hunk 2]
```diff
@@ -1,32 +1,35 @@
...
-List driverDetailsList = [];
-TransporterIdController tIdController = Get.find<TransporterIdController>();
-String driverApiUrl =
-    "http://ec2-15-207-113-71.ap-south-1.compute.amazonaws.com:9080/driver";
+Future<List> getDriverDetailsFromDriverApi(
+    BuildContext context, driverIdList) async {
+  var providerData = Provider.of<ProviderData>(context, listen: false);
+  var jsonData;
+  String driverApiUrl =
+      "http://ec2-15-207-113-71.ap-south-1.compute.amazonaws.com:9080/driver";
```
Comment (liveasy97) [inline lib/functions/getDriverDetailsFromDriverApi.dart:13]: use this from env file

File: lib/functions/getLoadPosterDetailsFromApi.dart [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+      jsonData = json.decode(response.body);
+
+      LoadPosterModel loadPosterModel = LoadPosterModel();
+      loadPosterModel.loadPosterId = jsonData["id"].toString();
+      loadPosterModel.loadPosterName = jsonData["name"].toString();
+      loadPosterModel.loadPosterCompanyName =
+          jsonData["companyName"].toString();
+      loadPosterModel.loadPosterPhoneNo = jsonData["phoneNo"];
+      loadPosterModel.loadPosterKyc = jsonData["kyc"].toString();
+      loadPosterModel.loadPosterApproved = jsonData["approved"].toString();
```
Comment (liveasy97) [inline lib/functions/getLoadPosterDetailsFromApi.dart:49]: there were few changes in shipperApi, so make changes accordingly

File: lib/screens/findLoadScreen.dart [hunk 4]
```diff
@@ -28,18 +29,12 @@ class _FindLoadScreenState extends State<FindLoadScreen> {
   @override
   Widget build(BuildContext context) {
     var providerData = Provider.of<ProviderData>(context,listen: false);
-    if (Provider.of<ProviderData>(context).loadingPointCity != "") {
```
Comment (liveasy97) [inline lib/screens/findLoadScreen.dart:31]: maybe you removed this by mistake. Make it as it was

File: lib/widgets/alertDialog/bidButtonAlertDialog.dart [hunk 5]
```diff
@@ -7,110 +7,210 @@ import 'package:liveasy/constants/spaces.dart';
...
+        )
+      ],
+      shape: RoundedRectangleBorder(
+        borderRadius: BorderRadius.all(Radius.circular(radius_2 - 2)),
+      ),
+    );
+  }
 }
+
+// Future<void> showInformationDialog(BuildContext context, loadId) async {
```
Comment (liveasy97) [inline lib/widgets/alertDialog/bidButtonAlertDialog.dart:118]: remove commented code and print statements if not needed

File: lib/widgets/buttons/verifyNowButton.dart [hunk 6]
```diff
@@ -5,13 +5,14 @@ import 'package:liveasy/constants/fontSize.dart';
...
 import 'package:liveasy/constants/spaces.dart';
+import 'package:liveasy/screens/accountScreens/accountVerificationPage1.dart';
 class VerifyNowButton extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
     return GestureDetector(
       onTap: () {
-        Get.back();
+        Get.to(()=> AccountVerificationPage1());
```
Comment (liveasy97) [inline lib/widgets/buttons/verifyNowButton.dart:15]: don't do this. Rather, use Get.to(NavigationScreen()); and Provider.of<ProviderData>(context, listen:false).updateIndex(4);

File: lib/widgets/cardLowerWidget.dart [hunk 7]
```diff
@@ -1,10 +1,15 @@
-//container below card
 import 'package:flutter/material.dart';
 import 'package:liveasy/constants/fontSize.dart';
 import 'package:liveasy/constants/fontWeights.dart';
 import 'package:liveasy/widgets/buttons/callButton.dart';
 
-class ContactWidget extends StatelessWidget {
+// ignore: must_be_immutable
+class CardLowerWidget extends StatelessWidget {
```
Comment (liveasy97) [inline lib/widgets/cardLowerWidget.dart:7]: change name to more descriptive like LoadCardFooter or so

File: lib/widgets/displayLoadsCard.dart [hunk 8]
```diff
@@ -0,0 +1,133 @@
...
+  TransporterIdController tIdController = Get.find<TransporterIdController>();
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(children: [
+      GestureDetector(
+        onTap: () {
+          // ignore: unrelated_type_equality_checks
+          if (tIdController.transporterApproved == true) {
+            Get.to(() => LoadDetailsScreen(
```
Comment (liveasy97) [inline lib/widgets/displayLoadsCard.dart:79]: rather then passing so many details in constructor, pass a model

File: lib/widgets/selectTruckDropDownWidget.dart [hunk 9]
```diff
@@ -0,0 +1,111 @@
+// import 'package:flutter/material.dart';
```
Comment (liveasy97) [inline lib/widgets/selectTruckDropDownWidget.dart:1]: delete such files if not needed

File: lib/functions/getDataFromApi.dart (hunk 1 above)
Comment (Shikhar-Pandey) [reply lib/functions/getDataFromApi.dart:5]: No use of this file, deleted it

File: lib/widgets/buttons/verifyNowButton.dart (hunk 6 above)
Comment (Shikhar-Pandey) [reply lib/widgets/buttons/verifyNowButton.dart:15]: Provider.of(context, listen:false).updateIndex(4);
where to use this line?

Comment (liveasy97) [reply lib/widgets/buttons/verifyNowButton.dart:15]: in the onTap
it will take you to Navigation Screen with account button selected

**Pull request merged**